lib/config.js

```diff
@@ -177,6 +181,18 @@ const peva_province = [
   "http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/41caf7e6-b040-4720-9cc2-a96cfffed5b4", // Leidend Ambtenaar
 ];
 
+const associationOther = [
+  "http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/013cc838-173a-4657-b1ae-b00c048df943", // Raad van bestuur
+  "http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/b52094ff-21a2-4da8-8dbe-f513365d1528", // Algemene vergadering
+  "http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/41caf7e6-b040-4720-9cc2-a96cfffed5b4", // Leidend Ambtenaar
+];
+
+const corporationOther = [
+  "http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/013cc838-173a-4657-b1ae-b00c048df943", // Raad van bestuur
+  "http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/b52094ff-21a2-4da8-8dbe-f513365d1528", // Algemene vergadering
+  "http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/41caf7e6-b040-4720-9cc2-a96cfffed5b4", // Leidend Ambtenaar
+];
```

Comment:
**Comment:** There is a lot of URI duplication in the file, which is error-prone. This is error prone. Maybe good idea to create a follow-up task to move URIs to a constants file and use these constants in array for better maintainability.

Reply:
Follow up issue created.